src/pages/MyContent/CharacterSheet/CharacterSheetCustomFeatures.vue

```diff
@@ -1,14 +1,26 @@
 <script lang="ts">
   import { Component, Prop, Vue } from 'vue-property-decorator'
+  import Draggable from 'vuedraggable'
   import MyDialog from '@/components/MyDialog.vue'
   import TextEditor from '@/components/TextEditor.vue'
   import ConfirmDelete from '@/components/ConfirmDelete.vue'
 
   @Component({
     components: {
+      Draggable,
       MyDialog,
       TextEditor,
       ConfirmDelete
+    },
+    computed: {
+      draggableFeatures: {
+        get () {
+          return this.$props.features
+        },
+        set (value) {
+          this.$emit('updateCharacter', { customFeatures: value })
+        }
+      }
```

Comment:
```suggestion
    get draggableFeatures () {
      return this.features
    }

    set draggableFeatures (customFeatures) {
      this.$emit('updateCharacter', { customFeatures })
    }
```
You can define getters and setters inside of the class component instead of in the component decorator. 
https://class-component.vuejs.org/guide/class-component.html#computed-properties